doc/install/install-ceph-gateway.rst

```diff
@@ -145,6 +144,28 @@ execute the following as the ``root`` user::
 
  iptables-save > /etc/iptables/rules.v4
 
+Using SSL with Civetweb
+-----------------------
+.. _Using SSL with Civetweb:
+
+Before using SSL with civetweb, you will need a certificate that will match
+the host name that that will be used to access the Ceph Object Gateway.
+You may wish to obtain one that has `subject alternate name` fields for
+more flexibility.  If you intend to use S3-style subdomains
+(`Add Wildcard to DNS`_), you will need a `wildcard` certificate.
+
+Civetweb requires that the server key, server certificate, and any other
+CA or intermediate certificates be supplied in one file.  Each of these
+items must be in `pem` form.  Because the combined file contains the
+secret key, it should be protected from unauthorized access.
```

Comment:
Should we clarify "unauthorized access" here? Explicitly "Only the UID that runs radosgw (the "ceph" UID) should have read access to this key file on the RGW server" ?